.yarn/versions/4aa0961e.yml

```diff
@@ -0,0 +1,34 @@
+releases:
+  "@yarnpkg/core": patch
+  "@yarnpkg/plugin-workspace-tools": patch
```

Comment:
I'm not sure if modifying the console output of `foreach` would be considered a breaking change?

Reply:
No, if the output is supposed to be used by tools (and thus be stable) we would have had a `--json` flag.